### PR TITLE
Add chillaranand to Appknox team

### PR DESCRIPTION
**Pull request by ChillarAnand**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/appknox/appknox.github.io/pull/10" target="_blank">ReviewNinja

File: _data/members.yml [hunk 1]
```diff
@@ -168,3 +168,13 @@
...
   github:
   www:
+
+- name: ChillarAnand
+  title: Backend Developer
+  gravatar: https://www.gravatar.com/avatar/45a742e07ab5f1e1944cef0701ff0c4b.png?s=150
+  linkedIn:
+  facebook:
+  www: http://avilpage.com/
+  email: anand21nanda@gmail.com
```
Comment (subho007) [inline _data/members.yml]: @ChillarAnand you can use your Appknox email id if you want.

**Comment by subho007** [comment]
@ChillarAnand resolve the conflict and send in another PR 👍

**Comment by ChillarAnand** [comment]
@subho007 Resolved conflict.

File: _data/members.yml [hunk 2]
```diff
@@ -196,3 +196,13 @@
   twitter: https://twitter.com/kartikjha
   facebook: https://www.facebook.com/kajh92
   email: kartik@appknox.com
+
+- name: ChillarAnand
+  title: Backend Developer
+  gravatar: https://www.gravatar.com/avatar/45a742e07ab5f1e1944cef0701ff0c4b.png?s=150
```
Comment (subho007) [inline _data/members.yml:202]: @ChillarAnand that is @dhilipsiva gravatar. Can you paste in your gravatar URL ? see http://appknox.github.io/company/#team 😄

**Pull request merged**